src/sentry/static/sentry/app/utils/fetchSentryAppComponents.jsx

```diff
@@ -0,0 +1,13 @@
+import SentryAppComponentsStore from 'app/stores/sentryAppComponentsStore';
+
+const fetchSentryAppComponents = (api, orgSlug, projectId) => {
```

Comment:
This should be an action creator, and you should fire actions that the stores listen to. See `app/actionCreators/savedSearches`